### DISCUSSION TITLE
Welcome to ngx-konva discussions!

### DISCUSSION THREAD

**Discussion by ghost** [giovanni-bertoncelli/ngx-konva#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by clabnet** [comment]
Hi, this is Claudio from Turin, Italy. 
I'm a senior web engineer, Vue, NodeJS, Angular, Typescript experiences.
First time with Konva. Nice to meet you Giovanni!
